akka-docs/src/main/paradox/java/multi-jvm-testing.md

```diff
@@ -0,0 +1 @@
+../scala/multi-jvm-testing.md
```

Comment:
Ah it's because there's a reference to `multi-jvm-testing` in `cluster-usage.md`, wrapped in a `@@@ div { .group-scala }`. Makes total sense to add this symlink.